data/json/monster_factions.json

```diff
@@ -2,6 +2,7 @@
     {
         "type" : "MONSTER_FACTION",
         "name" : "",
+        "base_faction" : "",
```

Comment:
Isn't that a bit pointless? The default for this is an empty string anyway `jo.get_string( "base_faction", "" )` and it's never checked whether the entry exists at all. Changing this to an empty string in json shouldn't be recognized by the code at all.